content/en/docs/reference/using-api/api-overview.md

```diff
@@ -18,95 +18,115 @@ This page provides an overview of the Kubernetes API.
-  as part of the `apiVersion` field, for example, `apiVersion: v1`.
-- The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use `apiVersion: $GROUP_NAME/$VERSION`
-  (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in
-  [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
+*  The *core* (also called *legacy*) group is found at REST path `/api/v1`.
+   The core group is not specified as part of the `apiVersion` field, for
+   example, `apiVersion: v1`.
+*  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use
+   `apiVersion: $GROUP_NAME/$VERSION` (for example, `apiVersion: batch/v1`).
+   You can find the full list of supported API groups in
+   [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
 
-The two paths that support extending the API with [custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
 
-- [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
-  for basic CRUD needs.
-- [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md) for a full set of Kubernetes API semantics to implement their own API server.
+## API Extension
 
-## Enabling or disabling API groups
+The two paths that support extending the API with
+[custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
 
-Certain resources and API groups are enabled by default. You can enable or disable them by
-setting `--runtime-config` on the API server. The `--runtime-config` flag accepts comma separated
-key=value pairs describing the runtime configuration of the API server. For example:
+1. [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
+   lets you declaratively define how the API server should provide your chosen resource API.
+1. You can also [implement your own extension API server](/docs/tasks/extend-kubernetes/setup-extension-api-server/)
+   and use the [aggregator](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
+   to make it seamless for clients.
 
-- to disable batch/v1, set `--runtime-config=batch/v1=false`
-- to enable batch/v2alpha1, set `--runtime-config=batch/v2alpha1`
+## Enabling or disabling API groups   {#enabling-or-disabling}
+
+Certain resources and API groups are enabled by default. You can enable or
+disable them by setting `--runtime-config` on the API server.  The
+`--runtime-config` flag accepts comma separated `<key>=<value>` pairs
+describing the runtime configuration of the API server. For example:
+
+ - to disable `batch/v1`, set `--runtime-config=batch/v1=false`
+ - to enable `batch/v2alpha1`, set `--runtime-config=batch/v2alpha1`
+
+The flag accepts comma separated set of key=value pairs describing runtime
```

Comment:
I think I cleaned this up. Would you agree that lines 114-115 repeat line 107-109?

Reply:
Agreed. Will drop this paragraph.
BTW, when resolving conflicts reported by git, it is really difficult to think clearly.